scaladoc-js/src/content-contributors/ContentContributors.scala

```diff
@@ -0,0 +1,79 @@
+package dotty.tools.scaladoc
+
+import org.scalajs.dom._
+import org.scalajs.dom.ext._
+
+import scala.util.matching.Regex._
+import scala.util.matching._
+import org.scalajs.dom.ext.Ajax
+import scala.scalajs.js.JSON
+import scala.scalajs.js
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Success,Failure}
+
+// Contributors widget
+// see https://stackoverflow.com/a/19200303/4496364
+// Copied from https://github.com/scala/docs.scala-lang/blob/main/resources/js/functions.js and rewritten to Scala.js
+
+case class FullAuthor(name: String, url: String, image: String)
+
+trait FallbackAuthor extends js.Object:
+  def name: String
+
+trait CommitBottom extends js.Object:
+  def author: FallbackAuthor
+
+trait Author extends js.Object:
+  def login: String
+  def avatar_url: String
+  def html_url: String
+
+trait CommitTop extends js.Object:
+  def commit: CommitBottom
+  def author: Author
+
+trait Commits extends js.Array[CommitTop]
+
+class ContentContributors:
```

Comment:
Actually, it works for every library that is hosed on GitHub and has correctly set up `source-links`. No more configuration is needed

Reply:
Oh, I missed your point here. As it comes to having it as an external implementation, the only possible solution is to have release-agnostic small js script that would be hosted on some stable domain (github release assets) and would be fetched each time and then executed. Or maybe you have another idea how we could inject dynamically that component?

Reply:
I wonder if it could be part of the Scala 3 website assets without having to be part of scaladoc itself?

Reply:
All the assets are packaged with the scaladoc, therefore you cannot update the assets without releasing a new scaladoc. The only way to be scaladoc agnostic, the script would have to be hosted somewhere else with stable URI and would be fetched on js runtime.

Reply:
It is possible to add static resources to a scaladoc-generated site: https://docs.scala-lang.org/scala3/guides/scaladoc/static-site.html#static-resources

We could rely on that to publish the .js files along with the Scala 3 website (without having to bake the feature in scaladoc itself).

Reply:
I am not sure if it will be an easy to implement, though `extraJS` property in templates should be sufficient to pass explicitly js to be loaded. Will try that then.

Reply:
I checked that, the `js` script should land under `docs/js` directory in dotty repo and then we would set up `- extraJS: js/contributors.js` in the frontmatter